src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+const Big = require('big.js');
+
```

Comment:
use `import`, `"require"` is not pure ES6 syntax
```
import Big from 'big.js';
```

Reply:
Done!!! :+1: